.circleci/config.yml

```diff
@@ -91,15 +91,18 @@ commands:
     steps:
       - restore_cache:
           name: Restore npm cache
-          key: v1-npm-{{ checksum "package.json" }}-{{ arch }}
+          key: v2-npm-{{ checksum "package.json" }}-{{ arch }}
       - run:
           name: Getting JavaScript dependencies
-          command: NODE_ENV=development npm install
+          command: NODE_ENV=development npm install --ignore-scripts
```

Comment:
Needed to install the node modules without them being modified by the postinstall script and then running post-install after the vanilla node_modules has been saved to the cache